### DISCUSSION TITLE
Tactile Sensor Intergration

### DISCUSSION THREAD

**Discussion by Yiju1213** [isaac-sim/IsaacLab#977] [General]
Is there anyway the isaac lab can integrate tactile sensors?

**Comment by zahir-w** [comment]
Hi, I noticed the orbit (isaaclab) paper on RAL2023 said that the further enhancement of the framework includes the tactile sensor integration. Maybe we need wait for next release?

#322

**Reply by Yiju1213** [reply]
Hello friend, I just check the paper related. It seems that we need to wait for future progress for sure haha. Many thanks for answering!
